website_crm_quick_answer/migrations/12.0.1.1.0/post-migration.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 Tecnativa - Jairo Llopis
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate(use_env=True)
+def migrate(env):
+    # The template is noupdate=1; force lang update
+    template = env.ref("website_crm_quick_answer.email_template")
```

Comment:
Perform this with the standard load of noupdate_changes.xml

Reply:
I didn't have a good experience with that the last time, and it's gonna do the same. Can't we just keep it as it is? :roll_eyes:

Reply:
Why you didn't have? As said, it's the standard way of doing this kind of stuff, because:

1. You have data in a separate file for its analysis.
2. Only one line of migration script is required.

Reply:
```
➤ python $openupgrade/scripts/compare_noupdate_xml_records.py website_crm_quick_answer-old website_crm_quick_answer --mode module


website_crm_quick_answer-old:

No differences.
```

![](https://media.giphy.com/media/fxNBiNEK7vl0J9aM7L/giphy.gif)

Reply:
Are you using the script from latest version 13.0?